### ISSUE TITLE
No Waypoints Being Made

### ISSUE THREAD

**Issue by Suzwyl** [lack/way2wayfabric#7]
I tried adding this mod to my private server, but it doesn't seem to work in my current setup. The description said no config needed and it should sync automatically, however, no waypoints were made. Perhaps it no longer works with more recent updates from the other mods in its current state?

Setup Info
Game ver: 1.19.2
Modloader: Fabric 0.14.25
W2WFabric ver: -0.0.6
Xaero's Minimap ver: 23.9.3_Fabric_1.19.1
Xaero's Worldmap ver: 1.37.2_Fabric_1.19.1
Waystones (Fabric Edition) ver: 1.19.2-11.4.2

**Comment by ST-DDT** [comment]
I have the same issue.

- way2wayfabric-0.0.6
- wraith-waystones-3.3.2+mc1.20.1
- Xaeros_Minimap_23.9.7_Fabric_1.20
- XaerosWorldMap_1.37.7_Fabric_1.20

> [11:53:33] [Render thread/INFO]: Found compatible map mod [xaerominimap 23.9.7]
> [11:53:33] [Render thread/INFO]: Registered with FabricWaystones for waypoint sync
> [11:53:33] [Render thread/INFO]: Way2wayFabric has been initialized for 1 waystone provider(s)

**Comment by mwlol** [comment]
Yeah same problem happens here but it happens after I discovered 8 waypoints then it stopped working.
way2way fabric 0.0.6
XaerosWorldMap fabric 1.37.8

**Comment by mwlol** [comment]
> Yeah same problem happens here but it happens after I discovered 8 waypoints then it stopped working. way2way fabric 0.0.6 XaerosWorldMap fabric 1.37.8

Sorry and also fabric waystone 3.3.2+mc1.20.1

**Comment by lack** [comment]
Thanks!  I'll try to replicate this.  Any change you can send me the logs from your game, especially the one where it synced only the first 8 waystones?

**Comment by johndelker** [comment]
I'm also having this issue. Nothing in my log about way2wayfabric except that it loads. Strangely, they show up fine for other players on my server. We have a synced modpack so there shouldn't be any differences between our setup.
